Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -851,6 +851,32 @@ public ArraySegment<byte> GetPayloadBytes(FileAccessSetup setup, MemoryStream st
             }
         }
 
+        /// <summary>
+        /// Explicitly releases most of its memory.
+        /// </summary>
+        internal void Release()
+        {
+            m_normalizedFragments?.Clear();
+            var workList = new Stack<Node>();
+            workList.Push(m_rootNode);
+            while (workList.Any())

```

Comment:
>workList.Any() [](start = 19, length = 14)

Use `Count`. This will create possibly tons of `Enumerator`, and since it's called from interface, that enumerator gets boxed.

Reply:
changed